website/docs/r/pi_volume_attach.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 ---
 
 # ibm_pi_volume_attach
-Attaches volume to a Power Systems Virtual Server instance. For more information, about managing volume, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
```

Comment:
Wait Dipti I talked @michaelkad and we need a new resource to detach. I don't think we need this PR anymore.

@michaelkad could you confirm?

Reply:
@ismirlia I have looked back at it and we do not need another resource, since the destroy does the detach for this. The update is correct.